bootstrap/scss/_custom-styles.scss

```diff
@@ -83,6 +83,9 @@ footer {
     padding: 5rem; 
     background-color: #e9ecef;
     opacity: 0.7;
+    @include media-breakpoint-down(sm) {
+        padding: 1.25rem;
```

Comment:
It would be great if we can use the Sass map for spacing here, like in the other repositories: https://github.com/laminas/laminas.github.io/blob/28f0fbc81e3821ff12d8ae5fdcd6d6d2a2c088f7/asset/sass/_elements.scss#L41

Unfortunately this was not done during the initial creation.

Reply:
done